index.bs

```diff
@@ -6554,6 +6577,98 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheBypass Command ####  {#command-network-setCacheBypass}
+
+The <dfn export for=commands>network.setCacheBypass</dfn> command bypasses the
+network cache for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheBypass = (
+        method: "network.setCacheBypass",
+        params: network.SetCacheBypassParameters
+      )
+
+      network.SetCacheBypassParameters = {
+        bypass: bool,
+        ? contexts: [+browsingContext.BrowsingContext]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The <dfn export>WebDriver BiDi bypass cache</dfn> steps given |request| are:
```

Comment:
The fetch spec draft is not updated yet, are these steps going to be used there?